frontend/src/scene/main-view/CourseModal.js

```diff
@@ -104,7 +106,36 @@ const ReservationContent = styled(Content)`
     }
 `;
 
-const MainModal = ({ course, seletectedDate, onConfirm, clear }) => (
+const ErrorMessageAnimation = posed.h4({
+    hidden: {
+        y: -10,
+        x: 50,
+        opacity: 0,
+    },
+    shown: {
+        y: -10,
+        x: 0,
+        opacity: 1,
+    },
+});
+
+const ErrorMessageTag = styled(posed.div)`
+    color: ${(props) => props.theme[props.color]};
+    font-size: 2.3rem;
+    font-weight: bold;
+    margin: 0;
+    key="3"
+    color={errorDetail.colorCode}
+    style={{ marginTop: 10 }}
```

Comment:
`margin-Top : 1rem` , no need to use style tag here.

Reply:
It was from `                    <ErrorMessageTag
                        key="3"
                        color={errorDetail.colorCode}
                        style={{ marginTop: 10 }}` I did not change it. But I will update accordingly

Reply:
You need to check if 1rem is enough or not. if not change accordingly.